packages/datagateway-download/public/res/default.json

```diff
@@ -34,7 +34,11 @@
     "expired": "Expired",
     "paused": "Paused",
     "preparing": "Preparing",
-    "restoring": "Restoring from tape"
+    "restoring": "Restoring from tape",
+    "progress": "Progress",
+    "calculating_progress": "Calculating...",
+    "progress_unavailable": "Unavailable",
```

Comment:
A minor comment from yesterday was that `Unavailable` was confusing and they think perhaps `Unknown` is better. Might as well change the default before it merges in